### PR TITLE
Add the "Vaststelling gemeentelijk beleidskader (gemeentewegen) form

### PR DESCRIPTION
**Pull request by Windvis**
Related PR: https://github.com/lblod/manage-submission-form-tooling/pull/8

This also includes a small html fix for some help text contents to make it consistent with the version for the new form.

### CODE REVIEW

File: config/migrations/2022/20220119141600-new-form/20220119141700-add-new-besluit-type.ttl [hunk 1]
```diff
@@ -0,0 +1,15 @@
+INSERT DATA {
```
Comment (Windvis) [inline config/migrations/2022/20220119141600-new-form/20220119141700-add-new-besluit-type.ttl:1]: I had some issues with this migration and I still haven't figured it out. After the migration ran (the migration service marked it as done) the data wasn't actually added in the database which meant the form option wasn't visible. After copying this query in Virtuoso's SPARQL interface it did show work so it's not the query itself.

Comment (HugaertsDries) [reply config/migrations/2022/20220119141600-new-form/20220119141700-add-new-besluit-type.ttl]: @Windvis does this issue still persist? I think it's due to the file extension being defined as `.ttl` instead off `.sparql`. It is weird that the migrations service wouldn't complain, that I do admit.

Comment (Windvis) [reply config/migrations/2022/20220119141600-new-form/20220119141700-add-new-besluit-type.ttl]: Doh, that will be it for sure, thanks. I haven't retested it to be honest, I didn't create a backup of a "clean db" and running the migrations from 0 takes a very long time.

Comment (HugaertsDries) [reply config/migrations/2022/20220119141600-new-form/20220119141700-add-new-besluit-type.ttl]: :arrow_up: yup, that's seems to do the trick!

**Review by HugaertsDries — APPROVED** [review]
Form works as adventist, thumb-up from me!

File: config/migrations/2022/20220119141600-new-form/20220119141700-add-new-besluit-type.ttl (hunk 1 above)
Comment (Windvis) [reply config/migrations/2022/20220119141600-new-form/20220119141700-add-new-besluit-type.ttl]: Thanks!

**Pull request merged**